catalyst/dl/state/__init__.py

```diff
@@ -0,0 +1,4 @@
+# flake8: noqa
+# isort:skip_file
+
+from .gan import GanState
```

Comment:
also should be imported in catalyst/dl/__init__

Reply:
Done